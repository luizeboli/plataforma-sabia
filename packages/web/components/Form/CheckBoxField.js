import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const StyledCheckBox = styled.div`
	display: flex;
	flex-direction: row;
	align-items: center;
	padding: 1rem;
	font: 1.2rem;
`;

const StyledCheckBoxInput = styled.input`
	width: 2.5rem;
	height: 2.5rem;
	margin: 0rem;
	margin: 0 0.5rem 0 0;
`;
const StyledCheckBoxLabel = styled.div`
	width: 100%;
	color: ${({ theme }) => theme.colors.lightGray};
`;

const CheckBoxField = ({ name, value, label, required, onChange }) => {
	const handleOnChange = () => onChange((oldValue) => !oldValue);
	return (
		<StyledCheckBox>
			<StyledCheckBoxInput
				id={name}
				name={name}
				type="checkbox"
				aria-label={label}
				aria-required={required}
				required={required}
				checked={value}
				onChange={handleOnChange}
			/>
			<StyledCheckBoxLabel>{label}</StyledCheckBoxLabel>
		</StyledCheckBox>
	);
};
CheckBoxField.propTypes = {
	name: PropTypes.string.isRequired,
	value: PropTypes.bool,
	label: PropTypes.string,
	required: PropTypes.bool,
	onChange: PropTypes.func,
};

CheckBoxField.defaultProps = {
	value: false,
	label: '',
	required: false,
	onChange: () => {},
};

export default CheckBoxField;
