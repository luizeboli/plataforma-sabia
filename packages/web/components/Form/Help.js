import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';
import { AiFillQuestionCircle } from 'react-icons/ai';
import ReactTooltip from 'react-tooltip';

const Icon = styled(AiFillQuestionCircle)`
	width: 4rem;
	height: 4rem;
	margin: 0.6rem 0.5rem;
	fill: ${({ theme }) => theme.colors.mediumGray};
	transition: fill 0.2s;
	:hover {
		cursor: pointer;
		fill: ${({ theme }) => theme.colors.darkGray};
	}
`;

const Help = ({ id, HelpComponent }) => {
	return (
		<>
			<Icon data-tip data-for={id} />
			<ReactTooltip uuid="uuid" id={id} multiline type="dark" place="bottom" effect="solid">
				{HelpComponent}
			</ReactTooltip>
		</>
	);
};

Help.propTypes = {
	id: PropTypes.string,
	HelpComponent: PropTypes.node,
};

Help.defaultProps = {
	id: '',
	HelpComponent: <p>Help Text</p>,
};

export default Help;
