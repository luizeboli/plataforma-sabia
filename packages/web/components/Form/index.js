export { default as InputField } from './InputField';
export { default as SwitchField } from './SwitchField';
export { default as TextField } from './TextField';
export { default as SelectField } from './SelectField';
export { default as CheckBoxField } from './CheckBoxField';
export { default as MaskedInputField } from './MaskedInputField';
export { default as Watcher } from './Watcher';
export { default as InputHiddenField } from './InputHiddenField';
export { default as Repeater } from './Repeater';
export { default as Help } from './Help';
export * from './Form';
