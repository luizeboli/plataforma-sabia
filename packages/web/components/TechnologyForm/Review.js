import React from 'react';

const Review = () => {
	return (
		<h3 style={{ textAlign: 'center', marginBottom: '20px' }}>
			Sua tecnologia foi cadastrada com sucesso
		</h3>
	);
};

export default Review;
