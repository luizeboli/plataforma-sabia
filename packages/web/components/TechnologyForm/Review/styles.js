import styled, { css } from 'styled-components';
import * as Layout from '../../Common/Layout';
import { CheckBoxField } from '../../Form';
import * as MapAndAttachments from '../MapAndAttachments/styles';

export const Cell = styled(Layout.Cell)`
	width: 100%;
`;

export const Row = styled.div`
	display: flex;
	align-items: flex-start;

	& > *:not(:first-child):not(:last-child) {
		margin: 0 10px;
	}

	@media (max-width: ${({ theme }) => theme.screens.large}px) {
		flex-direction: column;
		margin-top: 1.5rem;
	}
`;

export const Wrapper = styled.div`
	margin-bottom: 4rem;
`;

export const Checkbox = styled(CheckBoxField)`
	div {
		text-transform: uppercase !important;
		a {
			border: 1px solid red;
			padding: 0;
		}
	}
`;

export const UploadsTitle = styled.span`
	${({ theme: { colors } }) => css`
		display: block;
		font-weight: 500;
		font-size: 1.4rem;
		margin-bottom: 1rem;
		text-transform: uppercase;
		color: ${colors.lightGray};

		&:not(:first-child) {
			margin-top: 1rem;
		}
	`}
`;

export const UploadedDocuments = styled(MapAndAttachments.UploadedDocuments)``;

export const UploadedImages = styled(MapAndAttachments.UploadedImages)``;

export const IconRow = styled(MapAndAttachments.IconRow)``;

export const Media = styled(MapAndAttachments.Media)``;

export const IconLink = styled(MapAndAttachments.IconLink)``;
