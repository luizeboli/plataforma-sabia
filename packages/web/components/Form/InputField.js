/* eslint-disable react/jsx-props-no-spreading */
import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';
import { useTranslation } from 'react-i18next';
import { InputFieldWrapper, InputLabel, InputError, Row } from './styles';
import { validationErrorMessage } from '../../utils/helper';
import Help from './Help';

const StyledInput = styled.input`
	width: 100%;
	height: 4.4rem;
	font-size: 1.4rem;
	margin: 0.5rem 0;
	padding: 1.2rem;
	background: white;
	border: 1px solid ${({ theme }) => theme.colors.mediumGray};
	border-radius: 0.2rem;
	color: ${({ theme }) => theme.colors.lightGray};
`;

const InputField = ({ name, form, type, label, help, validation, ...inputProps }) => {
	const { t } = useTranslation(['error']);
	const { register, errors } = form;

	return (
		<InputFieldWrapper hasError={typeof errors[name] !== 'undefined'}>
			<InputLabel htmlFor={name}>{label}</InputLabel>

			<Row>
				<StyledInput
					id={name}
					type={type}
					name={name}
					aria-label={label}
					aria-required={validation.required}
					ref={register(validation)}
					{...inputProps}
				/>
				{help && <Help id={name} HelpComponent={help} />}
			</Row>

			<InputError>{validationErrorMessage(errors[name], t)}</InputError>
		</InputFieldWrapper>
	);
};

InputField.propTypes = {
	name: PropTypes.string.isRequired,
	label: PropTypes.string,
	placeholder: PropTypes.string,
	icon: PropTypes.func,
	type: PropTypes.string,
	form: PropTypes.shape({
		register: PropTypes.func,
		errors: PropTypes.shape({}),
	}),
	help: PropTypes.node,
	/**
	 * @see https://react-hook-form.com/api#register
	 */
	validation: PropTypes.shape({
		required: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
	}),
};

InputField.defaultProps = {
	form: {},
	type: 'text',
	help: null,
	validation: {},
	label: '',
	placeholder: '',
	icon: () => false,
};

export default InputField;
