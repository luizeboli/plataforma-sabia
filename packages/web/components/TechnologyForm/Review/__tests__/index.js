import React from 'react';
import { render } from 'test-utils';
import Review from '..';
import { Form } from '../../../Form';
import { getFakeTechnology, normalizeAttachments } from '../../../../utils/technology';

let technology = getFakeTechnology();

technology = {
	...technology,
	attachments: normalizeAttachments(technology.attachments),
};

const data = {
	technology,
};

const onSubmit = jest.fn(() => {});

test('it render the review page', () => {
	const { container } = render(
		<Form onSubmit={onSubmit}>
			<Review data={data} />
		</Form>,
	);

	expect(container).toMatchSnapshot();
});
