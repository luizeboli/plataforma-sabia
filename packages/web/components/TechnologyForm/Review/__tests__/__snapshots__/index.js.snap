// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it render the review page 1`] = `
.c24 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 1rem;
  font: 1.2rem;
}

.c25 {
  width: 2.5rem;
  height: 2.5rem;
  margin: 0rem;
  margin: 0 0.5rem 0 0;
}

.c26 {
  width: 100%;
  color: hsla(0,0%,40%);
}

.c0 {
  margin: 1rem auto;
  padding: 2rem 1rem;
  width: 100%;
}

.c0 button,
.c0 a {
  padding-right: 3rem;
  padding-left: 3rem;
}

.c0 > label {
  width: 100%;
  display: block;
  font-weight: 700;
  font-size: 1.7rem;
}

.c4 {
  margin-left: 1rem;
  padding: 2rem 0;
}

.c5 {
  border-bottom: 4px solid hsla(0,0%,40%);
  width: calc(100% - 1rem);
  margin: 2rem 0;
}

.c5:first-child {
  margin-top: 0;
}

.c5 h4 {
  display: inline-block;
  background-color: hsla(0,0%,40%);
  color: hsla(0,0%,100%);
  padding: 2.5rem 3rem;
  font-weight: 600;
  font-size: 1.8rem;
  line-height: 2rem;
  text-transform: uppercase;
  margin-bottom: -4px;
}

.c20 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  margin-top: 0rem;
  margin-bottom: 1rem;
  background-color: transparent;
}

.c19 {
  padding-left: 2rem;
  margin: 1rem 0 0 0;
}

.c19 span {
  margin: 0 0 0 1rem;
}

.c19 svg {
  height: 2.5rem;
  width: 2.5rem;
}

.c6 {
  font-size: 1.6rem;
  line-height: 2.4rem;
}

.c6 strong {
  color: hsla(0,0%,7%);
}

.c6 span {
  color: hsla(204,7%,15%);
}

.c7 {
  width: 100%;
  height: 100%;
  min-height: 20rem;
  background: hsla(0,0%,100%);
  border: 0.1rem solid hsla(0,0%,80%);
  border-radius: 0.2rem;
  margin-top: 3rem;
}

.c7 section {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0.8rem;
  width: 100%;
  height: 100%;
  overflow: auto;
}

.c8 {
  font-size: 1.6rem;
  font-style: normal;
  font-weight: bold;
  background: hsla(0,0%,95%);
  width: 100%;
  padding: 1.2rem;
  text-transform: uppercase;
  color: hsla(204,7%,15%);
  border-bottom: 0.1rem solid hsl(0,0%,74%);
}

.c10 {
  border-collapse: collapse;
  width: 100%;
  color: hsla(204,7%,15%);
  overflow: scroll;
}

.c10 th,
.c10 td {
  padding: 0.8rem 0.5rem;
  border-bottom: 0.1rem solid hsl(0,0%,74%);
}

.c10 th:not(:first-child),
.c10 td:not(:first-child) {
  text-align: center;
}

.c10 tr:first-of-type {
  border-bottom: 0.2rem solid hsl(0,0%,74%);
}

.c10 tr:nth-child(even) {
  background-color: hsla(0,0%,95%);
}

.c10 tr th {
  font-weight: 500;
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  font-style: normal;
  font-weight: bold;
  font-size: 1.6rem;
  line-height: 2rem;
  padding: 2rem;
  color: hsla(204,70%,53%);
}

.c9 {
  color: hsla(204,7%,15%);
  text-align: center;
}

.c21 {
  width: 100%;
  height: 100%;
  background: hsla(0,0%,100%);
  margin-top: 3rem;
  background: transparent;
}

.c21 section {
  padding: 0.8rem;
  width: 100%;
  height: 100%;
  background: transparent;
  overflow-x: auto;
  overflow-y: hidden;
}

.c22 {
  border-collapse: collapse;
  width: 100%;
  color: hsla(204,7%,15%);
}

.c22 th,
.c22 td {
  padding: 0.8rem 0.5rem;
  border-bottom: 0.1rem solid hsl(0,0%,74%);
}

.c22 th:not(:first-child),
.c22 td:not(:first-child) {
  text-align: center;
}

.c22 thead th {
  font-weight: 500;
  border-bottom: 0.2rem solid hsl(0,0%,74%);
}

.c22 tbody tr:nth-child(odd) {
  background-color: hsl(0,0%,93%);
}

.c3 {
  -webkit-flex: 2;
  -ms-flex: 2;
  flex: 2;
  margin: 0 1rem 0 0;
  text-align: left;
  width: 100%;
}

.c23 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin: 0 1rem 0 0;
  text-align: left;
  width: 100%;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.c2 > *:not(:first-child):not(:last-child) {
  margin: 0 10px;
}

.c1 {
  margin-bottom: 4rem;
}

.c12 {
  display: block;
  font-weight: 500;
  font-size: 1.4rem;
  margin-bottom: 1rem;
  text-transform: uppercase;
  color: hsla(0,0%,40%);
}

.c12:not(:first-child) {
  margin-top: 1rem;
}

.c16 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c13 {
  display: grid;
  grid-template-columns: repeat(auto-fill,30%);
  grid-gap: 1rem;
}

.c14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c14 > * {
  margin-bottom: 1rem;
}

.c14 > a {
  padding: 0;
  margin-right: 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c17 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c17 > * {
  margin-bottom: 1rem;
}

.c17 > a {
  padding: 0;
  margin-right: 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c15 {
  width: 100%;
  -webkit-filter: opacity(0.7);
  filter: opacity(0.7);
}

.c18 {
  color: hsla(0,0%,7%);
}

@media (max-width:1170px) {
  .c20 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-align-items: stretch;
    -webkit-box-align: stretch;
    -ms-flex-align: stretch;
    align-items: stretch;
  }
}

@media (max-width:450px) {
  .c7 {
    max-width: 40rem;
  }
}

@media (max-width:1170px) {
  .c21 {
    max-width: 80rem;
  }
}

@media (max-width:768px) {
  .c21 {
    max-width: 65rem;
    margin: auto;
  }
}

@media (max-width:1170px) {
  .c3 {
    max-width: initial;
    margin: 0 0 1rem 0;
  }
}

@media (max-width:768px) {
  .c3 {
    margin: 0;
  }
}

@media (max-width:1170px) {
  .c23 {
    max-width: initial;
    margin: 0 0 1rem 0;
  }
}

@media (max-width:768px) {
  .c23 {
    margin: 0;
  }
}

@media (max-width:1170px) {
  .c2 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    margin-top: 1.5rem;
  }
}

@media (max-width:450px) {
  .c13 {
    grid-template-columns: repeat(auto-fill,100%);
  }
}

<div>
  <form
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="Layout__Cell-m6jxy6-4 c3"
        >
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Identificação
              </h4>
            </div>
            <div
              class="c6"
            >
              <strong>
                Título
                : 
              </strong>
              <span>
                Agebavzi niko zaro.
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Descrição
                : 
              </strong>
              <span>
                Gec hik gir.
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Categoria
                : 
              </strong>
              <span>
                Educação, Educação e calendário agrícola
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Classificação
                : 
              </strong>
              <span>
                Tecnologias Sociais
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Dimensão
                : 
              </strong>
              <span>
                Política
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Público-alvo
                : 
              </strong>
              <span>
                Agricultores
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Bioma
                : 
              </strong>
              <span>
                Caatinga
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Programa Governamental
                : 
              </strong>
              <span>
                Mais Nordeste
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Palavras-chave
                : 
              </strong>
              <span>
                Antelope, Python, Jaguarundi, Olive Sea Snake, Woodswallow
              </span>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Aspectos Legais
              </h4>
            </div>
            <div
              class="c6"
            >
              <strong>
                Tecnologia Patenteada
                : 
              </strong>
              <span>
                common:no
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Proteção Intelectual
                : 
              </strong>
              <span>
                Propriedade Intelectual 2
              </span>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Estágio de Desenvolvimento
              </h4>
            </div>
            <div
              class="c6"
            >
              <strong>
                Escala TRL
                : 
              </strong>
              <span>
                Nível 8 - Sistema real desenvolvido e aprovado
              </span>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Financiamento
              </h4>
            </div>
            <div
              class="c6"
            >
              <strong>
                Necessita de Financiamento
                : 
              </strong>
              <span>
                common:yes
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Tipo de Financiamento
                : 
              </strong>
              <span>
                Coletivo
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Valor do Financiamento
                : 
              </strong>
              <span>
                44546307
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Situação
                : 
              </strong>
              <span>
                Em aquisição
              </span>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Custos da Tecnologia
              </h4>
            </div>
            <div
              class="c7"
            >
              <div
                class="c8"
              >
                Custo de Desenvolvimento
              </div>
              <section>
                <p
                  class="c9"
                >
                  Nenhum custo cadastrado.
                </p>
              </section>
            </div>
            <div
              class="c7"
            >
              <div
                class="c8"
              >
                Custos de Implantação
              </div>
              <section>
                <table
                  class="c10"
                >
                  <thead>
                    <tr>
                      <th>
                        Descricao
                      </th>
                      <th>
                        Tipo
                      </th>
                      <th>
                        Qtde
                      </th>
                      <th>
                        Valor
                      </th>
                      <th>
                        Total
                      </th>
                    </tr>
                  </thead>
                  <tbody>
                    <tr>
                      <td>
                        Tofdezti bagavoh rutkorzum koba.
                      </td>
                      <td>
                        Equipamento
                      </td>
                      <td>
                        28
                      </td>
                      <td>
                        R$33,763,160.00
                      </td>
                      <td>
                        R$945,368,480.00
                      </td>
                    </tr>
                  </tbody>
                </table>
                <p
                  class="c11"
                >
                  R$945,368,480.00
                </p>
              </section>
            </div>
            <div
              class="c7"
            >
              <div
                class="c8"
              >
                Custos de Manutenção
              </div>
              <section>
                <p
                  class="c9"
                >
                  Nenhum custo cadastrado.
                </p>
              </section>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Documentos
              </h4>
            </div>
            <span
              class="c12"
            >
              Fotos da Tecnologia
            </span>
            <div
              class="styles__UploadedImages-sc-13jh09q-9 c13"
            >
              <div
                class="styles__IconRow-sc-13jh09q-15 c14"
              >
                <img
                  class="styles__Media-sc-13jh09q-14 c15"
                  src="http://127.0.0.1:3333/uploads/technologies/image.png"
                />
              </div>
            </div>
            <span
              class="c12"
            >
              Documentos
            </span>
            <div
              class="styles__UploadedDocuments-sc-13jh09q-8 c16"
            >
              <div
                class="styles__IconRow-sc-13jh09q-15 c17"
              >
                <a
                  class="styles__IconLink-sc-13jh09q-2 c18"
                  href="http://127.0.0.1:3333/uploads/technologies/image-1.pdf"
                >
                  <svg
                    fill="currentColor"
                    height="2rem"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 384 512"
                    width="2rem"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M181.9 256.1c-5-16-4.9-46.9-2-46.9 8.4 0 7.6 36.9 2 46.9zm-1.7 47.2c-7.7 20.2-17.3 43.3-28.4 62.7 18.3-7 39-17.2 62.9-21.9-12.7-9.6-24.9-23.4-34.5-40.8zM86.1 428.1c0 .8 13.2-5.4 34.9-40.2-6.7 6.3-29.1 24.5-34.9 40.2zM248 160h136v328c0 13.3-10.7 24-24 24H24c-13.3 0-24-10.7-24-24V24C0 10.7 10.7 0 24 0h200v136c0 13.2 10.8 24 24 24zm-8 171.8c-20-12.2-33.3-29-42.7-53.8 4.5-18.5 11.6-46.6 6.2-64.2-4.7-29.4-42.4-26.5-47.8-6.8-5 18.3-.4 44.1 8.1 77-11.6 27.6-28.7 64.6-40.8 85.8-.1 0-.1.1-.2.1-27.1 13.9-73.6 44.5-54.5 68 5.6 6.9 16 10 21.5 10 17.9 0 35.7-18 61.1-61.8 25.8-8.5 54.1-19.1 79-23.2 21.7 11.8 47.1 19.5 64 19.5 29.2 0 31.2-32 19.7-43.4-13.9-13.6-54.3-9.7-73.6-7.2zM377 105L279 7c-4.5-4.5-10.6-7-17-7h-6v128h128v-6.1c0-6.3-2.5-12.4-7-16.9zm-74.1 255.3c4.1-2.7-2.5-11.9-42.8-9 37.1 15.8 42.8 9 42.8 9z"
                    />
                  </svg>
                   
                  image-1.pdf
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="Layout__Cell-m6jxy6-4 c3"
        >
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Objetivos
              </h4>
            </div>
            <div
              class="c6"
            >
              <strong>
                Objetivo Principal
                : 
              </strong>
              <span>
                Rirut lavdes safbuz.
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Objetivos secundários
                : 
              </strong>
              <div
                class="c19"
              >
                <div
                  class="c20"
                >
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M504 256C504 119 393 8 256 8S8 119 8 256s111 248 248 248 248-111 248-248zm-448 0c0-110.5 89.5-200 200-200s200 89.5 200 200-89.5 200-200 200S56 366.5 56 256zm72 20v-40c0-6.6 5.4-12 12-12h116v-67c0-10.7 12.9-16 20.5-8.5l99 99c4.7 4.7 4.7 12.3 0 17l-99 99c-7.6 7.6-20.5 2.2-20.5-8.5v-67H140c-6.6 0-12-5.4-12-12z"
                    />
                  </svg>
                  <span>
                    Fet
                  </span>
                </div>
              </div>
              <div
                class="c19"
              >
                <div
                  class="c20"
                >
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M504 256C504 119 393 8 256 8S8 119 8 256s111 248 248 248 248-111 248-248zm-448 0c0-110.5 89.5-200 200-200s200 89.5 200 200-89.5 200-200 200S56 366.5 56 256zm72 20v-40c0-6.6 5.4-12 12-12h116v-67c0-10.7 12.9-16 20.5-8.5l99 99c4.7 4.7 4.7 12.3 0 17l-99 99c-7.6 7.6-20.5 2.2-20.5-8.5v-67H140c-6.6 0-12-5.4-12-12z"
                    />
                  </svg>
                  <span>
                    ejhagi
                  </span>
                </div>
              </div>
              <div
                class="c19"
              >
                <div
                  class="c20"
                >
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M504 256C504 119 393 8 256 8S8 119 8 256s111 248 248 248 248-111 248-248zm-448 0c0-110.5 89.5-200 200-200s200 89.5 200 200-89.5 200-200 200S56 366.5 56 256zm72 20v-40c0-6.6 5.4-12 12-12h116v-67c0-10.7 12.9-16 20.5-8.5l99 99c4.7 4.7 4.7 12.3 0 17l-99 99c-7.6 7.6-20.5 2.2-20.5-8.5v-67H140c-6.6 0-12-5.4-12-12z"
                    />
                  </svg>
                  <span>
                    pa
                  </span>
                </div>
              </div>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Problematização
              </h4>
            </div>
            <div
              class="c6"
            >
              <strong>
                Problemas que a tecnologia soluciona
                : 
              </strong>
              <span>
                Umfecet kig nepda.
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Problemas que a tecnologia acarreta
                : 
              </strong>
              <span>
                Jeit unape ni.
              </span>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Aplicação
              </h4>
            </div>
            <div
              class="c6"
            >
              <strong>
                Forma de Aplicação
                : 
              </strong>
              <span>
                Ci apa wovjejoru.
              </span>
            </div>
            <div
              class="c6"
            >
              <strong>
                Exemplos de Aplicação
                : 
              </strong>
              <span>
                Pumasaz sojkowi bikitmus.
              </span>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Contribuição
              </h4>
            </div>
            <div
              class="c6"
            >
              <strong>
                Contribuição para o semiárido
                : 
              </strong>
              <span>
                Gaoza nudta buru.
              </span>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Riscos
              </h4>
            </div>
            <div
              class="c6"
            >
              <strong>
                Riscos associados à tecnologia
                : 
              </strong>
              <span>
                Ni kurbopriv zuwcot.
              </span>
            </div>
          </div>
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Responsáveis
              </h4>
            </div>
            <div
              class="c21"
            >
              <section>
                <table
                  class="c22"
                >
                  <thead>
                    <tr>
                      <th>
                        Nome
                      </th>
                      <th>
                        E-mail
                      </th>
                      <th>
                        Telefone
                      </th>
                      <th>
                        ID Lattes
                      </th>
                      <th>
                        Cadastrado
                      </th>
                    </tr>
                  </thead>
                  <tbody>
                    <tr>
                      <td>
                        FirstName LastName
                      </td>
                      <td>
                        sabiatestinge2e@gmail.com
                      </td>
                      <td />
                      <td />
                      <td>
                        <svg
                          color="hsla(146, 64%, 36%)"
                          fill="currentColor"
                          height="3rem"
                          stroke="currentColor"
                          stroke-width="0"
                          style="color: rgb(33, 33, 33);"
                          viewBox="0 0 1024 1024"
                          width="3rem"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M699 353h-46.9c-10.2 0-19.9 4.9-25.9 13.3L469 584.3l-71.2-98.8c-6-8.3-15.6-13.3-25.9-13.3H325c-6.5 0-10.3 7.4-6.5 12.7l124.6 172.8a31.8 31.8 0 0 0 51.7 0l210.6-292c3.9-5.3.1-12.7-6.4-12.7z"
                          />
                          <path
                            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                          />
                        </svg>
                      </td>
                    </tr>
                    <tr>
                      <td>
                        FirstName LastName
                      </td>
                      <td>
                        sabiatesting@gmail.com
                      </td>
                      <td />
                      <td />
                      <td>
                        <svg
                          color="hsla(29, 73%, 56%)"
                          fill="currentColor"
                          height="3rem"
                          stroke="currentColor"
                          stroke-width="0"
                          style="color: rgb(61, 61, 61);"
                          viewBox="0 0 1024 1024"
                          width="3rem"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                          />
                          <path
                            d="M464 688a48 48 0 1 0 96 0 48 48 0 1 0-96 0zm24-112h48c4.4 0 8-3.6 8-8V296c0-4.4-3.6-8-8-8h-48c-4.4 0-8 3.6-8 8v272c0 4.4 3.6 8 8 8z"
                          />
                        </svg>
                      </td>
                    </tr>
                  </tbody>
                </table>
              </section>
            </div>
          </div>
        </div>
      </div>
      <div
        class="c2"
      >
        <div
          class="Layout__Cell-m6jxy6-4 c23"
        >
          <div
            class="c4"
          >
            <div
              class="c5"
              color="lightGray"
            >
              <h4>
                Termos de Aceitação
              </h4>
            </div>
            <div
              class="c24"
            >
              <input
                aria-label="Declaro que aceito os termos de uso"
                aria-required="true"
                class="c25"
                id="acceptUsageTerms"
                name="acceptUsageTerms"
                required=""
                type="checkbox"
              />
              <label
                class="c26"
                for="acceptUsageTerms"
              >
                Declaro que aceito os termos de uso
              </label>
            </div>
            <div
              class="c24"
            >
              <input
                aria-label="Declaro que aceito a política de privacidade"
                aria-required="true"
                class="c25"
                id="acceptPrivacyTerms"
                name="acceptPrivacyTerms"
                required=""
                type="checkbox"
              />
              <label
                class="c26"
                for="acceptPrivacyTerms"
              >
                Declaro que aceito a política de privacidade
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </form>
</div>
`;
